src/main.java

```diff
@@ -7,23 +7,36 @@ public class main {
 
 	LinkedList<String> listOfFiles = new LinkedList<String>();
 
+	String Path = "C:\\Users\\Gemst\\OneDrive\\autosync\\Dokument\\GitHub\\weatherGenerator\\";
```

Comment:
Hårdkodade paths är ajjabajja.
Kör istället typ:
```suggestion
	String Path = Paths.get(".").toAbsolutePath().toString() + "\\weatherGenerator\\;
        System.out.println("Executable Path: " + Path);
```

Reply:
True

Reply:
Fixat.